tests/run.sh

```diff
@@ -80,6 +80,10 @@ if [ "${1:-}" = "unit" ]; then
     docker exec -it elabtmp php vendor/bin/codecept run --skip api --skip apiv2 --coverage --coverage-html --coverage-xml
 elif [ "${1:-}" = "api" ]; then
     docker exec -it elabtmp php vendor/bin/codecept run --skip unit --coverage --coverage-html --coverage-xml
+# acceptance with cypress
```

Comment:
This should be made so it builds the container first if it doesn't exist.

Reply:
I added a test in the run.sh script [371e033](https://github.com/elabftw/elabftw/pull/4484/commits/371e03382501f5fd1c26118b1babefc5835c50dc)